texts/epn_paper.md

```diff
@@ -0,0 +1,72 @@
+# Let's take this discussion online
+
+<!-- #### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk. -->
+
+Your routine has changed drastically over the course of the last couple of months due to the lockdowns following the novel coronavirus outbreak.
+You now mostly stay at home, hopefully safe and healthy.
+Your classes are now taught online, and you probably had to quickly figure out how to run them at all.
+If you do experiments, your lab is either closed or operates at a much lower capacity.
+Your travel plans are cancelled. Still, despite all this, you have likely attended a scientific presentation or have even given one yourself.
+The only difference is that it happened online, perhaps as part of a virtual conference.
+
+<!-- #### Online talks were gaining momentum already before, and for a good reason. -->
+
+Virtual conferences were gaining momentum even before the coronavirus crisis due to multiple advantages over traditional in-person gatherings.
+With the increasing concerns about greenhouse emissions, many prefer to limit their long-distance travel.
+For some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether, and online events are the only available option.
+Finally, online events are free from most of the expensive and time-consuming aspects of conference organisation: finding a venue, arranging lodging and catering for the participants.
+With planning and logistics out of the way, virtual conferences can be organized much faster and respond to the latest developments—see a typical timeline in the figure.
+Because of being easier to attend and easily accommodating hundreds of participants, online events are more inclusive, and reach an even broader audience if the talks are recorded.
+
+![](planning_checklist.png)
+
+**Figure caption:** A conference organizer checklist.
+
+<!-- #### We decided to develop online conferences and established VSF. -->
+
+The idea of organizing virtual conferences brought us together—a group of geographically and academically diverse physics researchers.
+We started exploring the opportunities and challenges of online conferences in late Summer 2019, well before the COVID-19 pandemic.
+We decided to keep our organization open for anyone to join, and all of our documentation and notes were public from the very beginning, with the thought that these should benefit anyone trying to work towards the same goal.
+After several discussions we committed to developing a community platform for online conferences, which we now call the [Virtual Science Forum](https://virtualscienceforum.org).
+
+<!-- #### In thinking about online events we had to revisit many assumptions about conferences. -->
+
+Once we started brainstorming online events, we realized that some of our typical assumptions about conferences do not hold in a virtual format.
```

Comment:
```suggestion
Once we started brainstorming how to organize online events, we realized that a lot of usual assumptions about scientific meetings do not apply anymore.
Already the sizes of the events are very different: while a typical workshop will have anywhere between 50 and 100 participants, an online conference session can easily attract more than 500 attendees.
The duration of an online event is limited to a few hours because it needs to fit into the daily routines of participants spread across multiple hour zones—this in contrast to the median duration of a conference being a full working week.
```